finality-aleph/src/environment.rs

```diff
@@ -156,56 +156,35 @@ where
     pub(crate) fn new(
```

Comment:
The large number of arguments makes sad not only clippy but also myself. Maybe we should get rid of the constructor completely?

Reply:
I would keep new(), as there is a lot of fields to initialize and it would be a little cumbersome to do that from outside. At some point we can bundle together some of the arguments, but Environment is largely WIP at this point, so not sure it makes sense to do it now. I think we should turn off that warning from clippy :)

Reply:
Initializing the fields from the outside does not seem much more cumbersome than passing them to the constructor, and the constructor does nothing ;). But I am fine with leaving it as is. I will add an attribute to silence clippy